aws/src/test/java/org/apache/iceberg/aws/glue/TestIcebergToGlueConverter.java

```diff
@@ -40,6 +42,11 @@
 
 public class TestIcebergToGlueConverter {
 
+  private final Map<String, String> tableMetaDataProperties = ImmutableMap.of(
+      TableProperties.WRITE_DATA_LOCATION, "s3://writeDataLoc",
+      TableProperties.WRITE_METADATA_LOCATION, "s3://writeMetaDataLoc",
+      TableProperties.WRITE_FOLDER_STORAGE_LOCATION, "s3://writeFolderStorageLoc");
```

Comment:
Let's test a key will null values.

Reply:
I think, null as value is handled here we only have 3 keys ...  rather than [4](https://github.com/apache/iceberg/pull/4248/files#diff-b97b3e8bad418d1e6cdee1bc8b35c5db6f5df45b8d714c8640b51532bc86ecfbR68) 
4th Key i.e `TableProperties.OBJECT_STORE_PATH` is not present in this hashMap hence we will get Null when we do a hashMap lookup and it will be filtered out and not sent in additionalLocations.